### ISSUE TITLE
server: logging

### ISSUE THREAD

**Issue by blobject** [blobject/goose#2]
Implement a component that spits out info, warning, and/or debug messages produced by the server.

**Comment by Eloitor** [comment]
Idea: Info should be both human readable and valid tcl code. This allows working with logs without having to write a parser.

**Comment by blobject** [comment]
Yes, could be a chance for tcl to shine: print code as string and feed it directly as code.
